recipes/pytzer/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytzer" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pytzer-{{ version }}.tar.gz
+  sha256: 95149ae72814630348cfd83ba8bd4ef0677c5c6b7d01f2fa1245d40ccfec1069
+
+build:
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python